src/VisualStudio/Core/Impl/SolutionExplorer/BaseItem.cs

```diff
@@ -19,7 +19,7 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.SolutionExplore
 /// Microsoft.VisualStudio.Shell.TreeNavigation.HierarchyProvider.dll and
 /// Microsoft.VisualStudio.Shell.TreeNavigation.GraphProvider.dll.
 /// </summary>
-internal abstract class BaseItem :
+internal abstract class BaseItem(string name, bool canPreview = false) :
```

Comment:
added 'canPreview' support so you can just click on an item in the new solutionexplorersymbol results and have it navigate without having to double-click on it.